src/test/features/mvpFeature.test.js

```diff
@@ -0,0 +1,28 @@
+import App from '../../App'
+import React, {Component} from 'react';
+import {shallow, mount, render} from 'enzyme'
+
+
+describe('MVP feature', () => {
+  xit('You can choose a venue from list, see details and donate', () => {
```

Comment:
do we need to send this test to alice still if it is commented out? Or do we need to re-write it or re-enable it?